src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+  color: $main-text-color;
+  background-color: #fff;
+  border: 1px solid $light-grey-color;
+  padding: 32px 18px 16px;
+
+  &__img {
+    margin-bottom: 40px;
+    display: block;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-text-color;
```

Comment:
Ensure that the SCSS variable `$grey-text-color` is defined in your SCSS files. If not, it will cause a compilation error.